### DISCUSSION TITLE
Running Pocketbase as a Windows service on a Windows server

### DISCUSSION THREAD

**Discussion by GlutenBoy** [pocketbase/pocketbase#4863] [Q&A] [answered]
Hello,

I created a Windows service with the SC.exe command, this created the service just fine, but when I try to start the service I get this error:

```
[SC] StartService FAILED 1053:

The service did not respond to the start or control request in a timely fashion.
```

here's the command I used to create the service:
```
sc create sevenpacereporting binPath= "c:\timereporting\pocketbase\pocketbase.exe serve" DisplayName= "7pace Time Reporting" start= auto
```

Thank you!

**Comment by GlutenBoy** [comment, marked as answer]
FYI I fixed my issue by using this project: [pocketbase-service-windows](https://github.com/russkyc/pocketbase-service-windows)

**Reply by skpawar1305** [reply]
404 when trying to open https://github.com/russkyc/pocketbase-service-windows
